test/net/sourceforge/kolmafia/DataFileMechanicsTest.java

```diff
@@ -29,7 +29,7 @@ public static Stream<Arguments> data() {
         Arguments.of("buffbots.txt", 1, 3, 3),
         Arguments.of("cafe_booze.txt", 1, 2, 2),
         Arguments.of("cafe_food.txt", 1, 2, 2),
-        Arguments.of("classskills.txt", 4, 6, 7),
+        Arguments.of("classskills.txt", 5, 6, 7),
```

Comment:
Is this right?  I thought the arg was the version number of the version the test expected.  So it should be 6 now.  That raises the question - how long was this test expecting 4, getting version 5 and passing anyway?

Reply:
...good question

Reply:
`getVersionedReader` does its best to return a reader, so it looks like it was reading the file on the disk, finding that the version didn't match, then getting the one from the jar and using that. So hopefully everything worked out anyway.